apps/moderatorfeedback/serializers.py

```diff
@@ -34,7 +35,7 @@ def get_last_edit(self, feedback):
 
 class CommentWithFeedbackSerializer(a4_serializers.CommentSerializer):
     moderator_feedback = ModeratorCommentFeedbackSerializer(read_only=True)
-    ai_report = serializers.StringRelatedField(read_only=True)
+    ai_report = AiReportSerializer(read_only=True)
```

Comment:
@m4ra the `explanation` suffix was there and I forgot / was too lazy to change it. But I agree renaming it would probably be better. I can look into it tomorrow

Reply:
I changed it to `AiReport` now :)

Reply:
Thanx I am testing all together (a4 and defakts) locally now

Reply:
@goapunk functionality works well.
there are two thing regarding the design:
1. The XAI needs to be above the moderator's feedback in the userdashboard (according to design: https://app.zeplin.io/project/6492a63d7e162b416cfa77a0/screen/6492a6a4456e6a4829329054)

Now is under moderator's feedback, see screenshot:
![image](https://github.com/liqd/a4-defakts/assets/5695572/34dcc139-2497-4014-b244-0b6ff8134d0a)
 
2. In the moderator dashboard I see some  hardcoded text and a slider with arrows, which I don't see in the design:
![image](https://github.com/liqd/a4-defakts/assets/5695572/5c79f857-283c-4d47-8c4f-bd9c6f004600)

Reply:
@m4ra thanks, will fix 1. For 2. you can find it in the design here https://app.zeplin.io/project/6492a63d7e162b416cfa77a0/screen/64b6be8df2d9e45417f3cf09 (scroll down a bit). I think showing the real user reports was done in a different PR, so the slider should show them once merged